bucket/databricks_cli.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.215.0",
+    "description": "The Databricks command-line interface (also known as the Databricks CLI) provides a tool to automate the Databricks platform from your terminal, command prompt, or automation scripts.",
+    "homepage": "https://github.com/databricks/cli",
+    "license": {
+        "identifier": "Databricks",
+        "url": "https://raw.githubusercontent.com/databricks/cli/main/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/databricks/cli/releases/download/v0.215.0/databricks_cli_0.215.0_windows_amd64.zip",
+            "hash": "040ac9cd389f7b563141d1028efb805c2d0ec9499ece11c1267a0afa7f86f749"
+        }
+    },
+    "bin": "databricks.exe",
+    "shortcuts": [
+        [
+            "databricks.exe"
```

Comment:
The shortcut must contains two values : the exe and the "display name". you can take a look at the vscode.json.

That said, if it's only a CLI, it might not be relevant to have this in the windows menu (within scoop)... I'll let you the judge of that ;)

[Sorry for earlier, I didn't see this little issue while reviewing it on my phone ;)]